### PR TITLE
Add a task to create the GitHub release

### PR DESCRIPTION
**Pull request by FlorentClarret**
### What does this PR do?

Add a task to create the GitHub release

### Motivation

We currently do this manually and copy/paste it in the UI. 

We could include this in the CI one day if we think it makes sense. For now let's use it directly.

By default the release is a draft and not public so we can create it in advance and publish it only when we push the artefacts to the various registries

### Additional Notes

Usage: `inv/deva release.create-github-release 7.XX.Y`

### CODE REVIEW

File: tasks/release.py [hunk 1]
```diff
@@ -1049,3 +1049,55 @@ def create_qa_cards(ctx, tag):
...
+                lines.append(line)
+
+        # if we found the header, add the section to the final release note
+        if header_found:
+            notes.append(f"# {section}")
+            notes.extend(lines)
+
+    if not notes:
+        print(f"No release notes found for {version}")
+        return
```
Comment (CelianR) [inline tasks/release.py:1094]: **❓ question:** ‏Should this be an error / assert ?

**Review by KevinFairise2 — APPROVED** [review]

**Review by CelianR — APPROVED** [review]

File: tasks/release.py (hunk 1 above)
Comment (FlorentClarret) [reply tasks/release.py:1094]: adding it

File: tasks/release.py [hunk 2]
```diff
@@ -1049,3 +1049,55 @@ def create_qa_cards(ctx, tag):
...
+
+    sections = (
+        ("Agent", "CHANGELOG.rst"),
+        ("Datadog Cluster Agent", "CHANGELOG-DCA.rst"),
+    )
+
+    notes = []
+
+    for section, filename in sections:
+        text = pandoc.write(pandoc.read(file=filename), format="markdown_strict")
```
Comment (kacper-murzyn) [inline tasks/release.py:1069]: When running pandoc this way it creates the output with the wrapped lines, which look bad in the github release, so we pass `--wrap=none` when running this manually. Do you know if the result is the same when called from the code and if so - can we add such flag?

**Review by Pythyu — APPROVED** [review]
Aside from Kacper's comment LGTM

File: tasks/release.py (hunk 2 above)
Comment (FlorentClarret) [reply tasks/release.py:1069]: Ah true I forgot this one! Yes we can pass it. Thanks

**Review by kacper-murzyn — APPROVED** [review]

**Pull request merged**